lib/cisco_node_utils/cmd_ref/feature.yaml

```diff
@@ -74,3 +74,10 @@ vni:
     set_value: 'feature vn-segment-vlan-based'
   N8k: *feature_mt_lite
   N9k: *feature_mt_lite
+
+vtp:
+  kind: boolean
+  get_command: "show running vtp"
```

Comment:
Your right.. I meant to change it but just moved it over from our old vtp.yaml file.